src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/AuthenticationHelper.Digest.cs

```diff
@@ -415,12 +415,28 @@ public static void AppendKeyValue(this StringBuilder sb, string key, string valu
             if (includeQuotes)
             {
                 sb.Append('"');
+                int lastSpecialIndex = 0, specialIndex;
```

Comment:
What does the comma do? Just multiple declarations? I'd split it into 2 lines - less C# tricks in code IMO.